common/src/Microsoft.Azure.IIoT.Abstractions/src/Exceptions/MethodCallStatusException.cs

```diff
@@ -39,7 +39,7 @@ public class MethodCallStatusException : MethodCallException {
         /// <param name="errorMessage"></param>
         public MethodCallStatusException(string responsePayload, int result,
             string errorMessage = null) :
-            base($"Response {result} {errorMessage ?? ""}: {responsePayload}") {
+            base($"{{\"Message\":\"Response {result} {errorMessage ?? ""}\",\"Description\":{responsePayload}}}") {
             Result = result;
             ResponsePayload = responsePayload;
```

Comment:
Please note that `ResponsePayload` in the first constructor does not generate a proper JSON. We need to change that as `ResponsePayload` is picked up by `MethodRouter` to be the payload of direct method call response. So we need to fix that. Please see my previous comment about storing JSON payload in `Message` vs `ResponsePayload`, as one of the options would be to not overwrite `ResponsePayload` in the first constructor after it is being assigned here and make sure that we have a proper JSON here.

Reply:
yes changed